SourceX/storm/storm.cpp

```diff
@@ -41,10 +45,15 @@ void GetBasePath(char *buffer, size_t size)
 
 	snprintf(buffer, size, "%s", path);
 	SDL_free(path);
+#endif
 }
 
 void GetPrefPath(char *buffer, size_t size)
 {
+#if defined(__3DS__)
+	constexpr char path[] = "sdmc:/3ds/devilutionx/";

```

Comment:
`SDL_GetPrefPath` is already patched to return that.
Are the patches in this file necessary?

Reply:
I referenced a few early commits from other devilutionx ports to come up with this. With my limited developing / debugging experience, i'm eager to get corrected.

I haven't used devilutionx on any other platform but 3ds, and was under the impression multiple files had to be redirected.

Reply:
The point of SDL is to handle all the difference, so for the large majority, it should return the correct value for the platform. There are however a few exceptions. The early Switch port had a lot of unessesery exceptions so it can be a confusing reference.

Reply:
Looks like your using SDL1 but what you were referencing (probably Switch) used SDL2. What Glebm is pointing out is that you already patched the SDL1 version of `SDL_GetPrefPath` to return this path, so since your are using that there is no reason to overwrite it again here.